tests/imhttp-post-payload.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# This is part of the rsyslog testbench, licensed under ASL 2.0
+
+. ${srcdir:=.}/diag.sh init
+generate_conf
+IMHTTP_PORT="$(get_free_port)"
```

Comment:
that's the known issue - while this usually works OK, it is racy. There is no guarantee that the port will remain free.

A better solution, if possible, is:

* bind with port 0 
* make code ask OS which port it got
* persist that port to file
* make testbench pick port from there

The imtcp tests use that mode. But it depends on civitweb providing the necessary functionality.